--- mkvtoolnix-4.4.0.orig/share/desktop/mkvmergeGUI.desktop	2010-10-31 15:42:16.000000000 +0100
+++ mkvtoolnix-4.4.0/share/desktop/mkvmergeGUI.desktop	2010-10-31 20:04:07.076391913 +0100
@@ -7,7 +7,7 @@
 GenericName[fr]=Créateur de fichiers MKV
 GenericName[zh_TW]=MKV 檔製作軟體
 Icon=mkvmergeGUI
-Exec=mmg
+Exec=mkvmerge-gui
 Terminal=false
 MimeType=application/x-mmg-settings;video/x-matroska;audio/x-matroska;
 Categories=GNOME;AudioVideo;AudioVideoEditing;
