diff -ur mkvtoolnix-4.2.0.orig/share/desktop/mkvmergeGUI.desktop mkvtoolnix-4.2.0/share/desktop/mkvmergeGUI.desktop
--- mkvtoolnix-4.2.0.orig/share/desktop/mkvmergeGUI.desktop	2010-08-01 13:39:31.000000000 +0200
+++ mkvtoolnix-4.2.0/share/desktop/mkvmergeGUI.desktop	2010-08-01 14:21:45.176881240 +0200
@@ -6,7 +6,7 @@
 GenericName[es]=Creador de ficheros MKV
 GenericName[fr]=Créateur de fichiers MKV
 Icon=mkvmergeGUI
-Exec=mmg
+Exec=mkvmerge-gui
 Terminal=false
 MimeType=application/x-mmg-settings;video/x-matroska;audio/x-matroska;
 Categories=GNOME;AudioVideo;AudioVideoEditing;
