--- mkvtoolnix-1.5.6.orig/Makefile.in	2005-09-03 16:51:17.000000000 +0200
+++ mkvtoolnix-1.5.6/Makefile.in	2005-09-07 19:17:21.000000000 +0200
@@ -125,8 +125,8 @@
 		src/mkvextract@EXEEXT@ src/base64tool@EXEEXT@
 MANPAGES = doc/mkvmerge.1 doc/mkvinfo.1 doc/mkvextract.1 doc/base64tool.1
 ifeq (yes,$(USE_WXWINDOWS))
-APPLICATIONS += src/mmg/mmg@EXEEXT@
-MANPAGES += doc/mmg.1
+APPLICATIONS += src/mmg/mkvmerge-gui@EXEEXT@
+MANPAGES += doc/mkvmerge-gui.1
 endif
 
 TRANSLATIONS = @TRANSLATIONS@
@@ -141,6 +141,7 @@
 
 install-programs: $(APPLICATIONS)
 	$(mkinstalldirs) $(DESTDIR)$(bindir)
+	mv src/mmg/mmg@EXEEXT@ src/mmg/mkvmerge-gui@EXEEXT@
 	@list='$(APPLICATIONS)'; for p in $$list; do \
 	  p1=`echo $$p|sed 's/$(EXEEXT)$$//'`; \
 	  if test -f $$p ; then \
@@ -153,6 +154,7 @@
 
 install-mans:
 	$(mkinstalldirs) $(DESTDIR)$(man1dir)
+	mv doc/mmg.1 doc/mkvmerge-gui.1
 	@for i in $(MANPAGES); do \
 	  inst=`echo $$i | sed -e 's/^.*\///'`; \
 	  echo " $(INSTALL_DATA) $$i $(DESTDIR)$(man1dir)/$$inst"; \
@@ -461,6 +463,8 @@
 	@echo '    LINK ' $@
 	$(Q)$(LINK) -o $@ $(mmg_OBJECTS) $(mmg_LDADD)
 
+src/mmg/mkvmerge-gui@EXEEXT@: src/mmg/mmg@EXEEXT@
+
 #
 # include dependency files if they exist
 #
