diff -ur mkvtoolnix-1.7.0.orig/Makefile.in mkvtoolnix-1.7.0/Makefile.in
--- mkvtoolnix-1.7.0.orig/Makefile.in	2006-09-25 17:37:44.000000000 +0200
+++ mkvtoolnix-1.7.0/Makefile.in	2006-11-09 15:00:23.553384159 +0100
@@ -129,8 +129,8 @@
 		src/mkvextract@EXEEXT@ src/base64tool@EXEEXT@
 MANPAGES = doc/mkvmerge.1 doc/mkvinfo.1 doc/mkvextract.1 doc/base64tool.1
 ifeq (yes,$(USE_WXWIDGETS))
-APPLICATIONS += src/mmg/mmg@EXEEXT@
-MANPAGES += doc/mmg.1
+APPLICATIONS += src/mmg/mkvmerge-gui@EXEEXT@
+MANPAGES += doc/mkvmerge-gui.1
 endif
 
 TRANSLATIONS = @TRANSLATIONS@
@@ -145,6 +145,7 @@
 
 install-programs: $(APPLICATIONS)
 	$(mkinstalldirs) $(DESTDIR)$(bindir)
+	mv src/mmg/mmg@EXEEXT@ src/mmg/mkvmerge-gui@EXEEXT@
 	@list='$(APPLICATIONS)'; for p in $$list; do \
 	  p1=`echo $$p|sed 's/$(EXEEXT)$$//'`; \
 	  if test -f $$p ; then \
@@ -157,6 +158,7 @@
 
 install-mans:
 	$(mkinstalldirs) $(DESTDIR)$(man1dir)
+	mv doc/mmg.1 doc/mkvmerge-gui.1
 	@for i in $(MANPAGES); do \
 	  inst=`echo $$i | sed -e 's/^.*\///'`; \
 	  echo " $(INSTALL_DATA) $$i $(DESTDIR)$(man1dir)/$$inst"; \
@@ -466,6 +468,8 @@
 	@echo '    LINK ' $@
 	$(Q)$(LINK) -o $@ $(mmg_OBJECTS) $(mmg_LDADD)
 
+src/mmg/mkvmerge-gui@EXEEXT@: src/mmg/mmg@EXEEXT@
+
 #
 # include dependency files if they exist
 #
Only in mkvtoolnix-1.7.0: Makefile.in.orig
