diff -ur mkvtoolnix-4.0.0.orig/Makefile.in mkvtoolnix-4.0.0/Makefile.in
--- mkvtoolnix-4.0.0.orig/Makefile.in	2010-06-05 14:24:39.000000000 +0200
+++ mkvtoolnix-4.0.0/Makefile.in	2010-06-05 18:17:19.987445645 +0200
@@ -164,8 +164,8 @@
 APPLICATIONS = src/mkvmerge@EXEEXT@ src/mkvinfo@EXEEXT@ src/mkvextract@EXEEXT@ src/mkvpropedit@EXEEXT@
 MANPAGES = doc/man/mkvmerge.1 doc/man/mkvinfo.1 doc/man/mkvextract.1 doc/man/mkvpropedit.1
 ifeq (yes,$(USE_WXWIDGETS))
-APPLICATIONS += src/mmg/mmg@EXEEXT@
-MANPAGES += doc/man/mmg.1
+APPLICATIONS += src/mmg/mkvmerge-gui@EXEEXT@
+MANPAGES += doc/man/mkvmerge-gui.1
 endif
 
 TRANSLATIONS = @TRANSLATIONS@
@@ -250,6 +250,7 @@
 
 install-programs: $(APPLICATIONS)
 	$(mkinstalldirs) $(DESTDIR)$(bindir)
+	mv src/mmg/mmg@EXEEXT@ src/mmg/mkvmerge-gui@EXEEXT@
 	@list='$(APPLICATIONS)'; for p in $$list; do \
 	  p1=`echo $$p|sed 's/$(EXEEXT)$$//'`; \
 	  if test -f $$p ; then \
@@ -261,6 +262,7 @@
 
 install-mans:
 	$(mkinstalldirs) $(DESTDIR)$(man1dir)
+	mv doc/man/mmg.1 doc/man/mkvmerge-gui.1
 	@for i in $(MANPAGES); do \
 	  inst=`echo $$i | sed -e 's/^.*\///'`; \
 	  echo " $(INSTALL_DATA) $$i $(DESTDIR)$(man1dir)/$$inst"; \
@@ -269,6 +271,7 @@
 
 install-translated-mans:
 	@for lang in $(MANPAGES_TRANSLATIONS); do \
+	  mv doc/man/$$lang/mmg.1 doc/man/$$lang/mkvmerge-gui.1 ; \
 	  $(mkinstalldirs) $(DESTDIR)$(mandir)/$$lang/man1; \
 	  for manpage in $(MANPAGES); do \
 	    inst=`echo $$manpage | sed -e 's/^.*\///'`; \
@@ -632,6 +635,8 @@
 	@echo '    LINK ' $@
 	$(Q)$(LINK) -o $@ $(mmg_OBJECTS) $(mmg_LDADD)
 
+src/mmg/mkvmerge-gui@EXEEXT@: src/mmg/mmg@EXEEXT@
+
 .SECONDARY:
 
 #
Only in mkvtoolnix-4.0.0: Makefile.in.orig
