Description: Fix mkvinfo-gui's path in its .desktop file
Author: Moritz Bunkus <moritz@bunkus.org>
Last-Update: 2014-03-05
--- a/share/desktop/mkvinfo.desktop
+++ b/share/desktop/mkvinfo.desktop
@@ -7,7 +7,7 @@
 GenericName[zh_CN]=显示 MKV 文件的信息
 GenericName[zh_TW]=顯示 MKV 檔資訊
 Icon=mkvinfo
-Exec=mkvinfo -g %f
+Exec=mkvinfo-gui -g %f
 Terminal=false
 MimeType=video/x-matroska;audio/x-matroska;
 Categories=AudioVideo;AudioVideoEditing;
